css-canvas-medical/styles.css

```diff
@@ -0,0 +1,338 @@
+  color: white;
+  font-weight: normal;
+}
+
+.adjust-font-color-size-family {
+  color: white;
+  line-height: 1.6;
+  font-family: Avenir, sans-serif;
+  font-size: 15px;
+}
+
+.btn {
+  background-color: #294464;
+  color: white;
+  font-size: larger;
+  font-family:
+    Avenir,
+    sans-serif;
+  border-style: none;
+  padding: 15px;
+  margin-top: 30px;
+  border-radius: 5px;
+  width: 200px;
+}
+
+.backgroundcolor {
+  background-color: #fafafa;
+  margin-top: 40px;
+  padding-top: 40px;
+  padding-bottom: 60px;
+}
+
+.col-one-half {
+  width: 50%;
+}
+
+.add-font-family {
+  font-family:
+    Avenir,
+    sans-serif;
+}
+
+.align-content-center {
+  text-align: center;
+  margin: auto;
+  padding-bottom: 60px;
+}
+
+.col-two-fifth {
+  width: 40%;
+}
+
+.adjust-heading-font-color-size-family-weight {
+  margin-bottom: 15px;
+  color: #858585;
+  font-size: 25px;
+  font-weight: normal;
+  font-family:
+    "Tiempos Headline",
+    sans-serif;
+}
+
+.adjust-subheading-color-line-height {
+  line-height: 1.6;
+  color: #909090;
+}
+
+.adjust-font-weight-size-color-position {
+  text-align: center;
+  font-weight: normal;
+  font-size: 15px;
+  padding: 10px;
+  color: #909090;
+}
+
+.learn-more {
+  padding-top: 40px;
+  padding-bottom: 40px;
+  align-items: flex-start;
+}
+
+.align-content-left > h3 {
+  color: #858585;
+  font-size: 25px;
+  font-weight: normal;
+  font-family:
+    "Tiempos Headline",
+    sans-serif;
+}
+
+.align-content-left > p {
+  color: #858585;
+  font-size: 13px;
+}
+
+.align-content-right {
+  display: flex;
+  justify-content: flex-end;
+  text-align: right;
+}
+
+.btn-right {
+  background-color: #294464;
+  color: white;
+  font-size: larger;
+  border-style: none;
+  padding: 15px;
+  margin-top: 30px;
+  width: 200px;
+  border-radius: 5px;
+}
+
+.footer {
+  background-color: #525351;
+  padding-top: 80px;
+  padding-bottom: 80px;
+}
+
+.left-content {
+  display: flex;
+  flex-direction: column;
+  color: white;
+  line-height: 1.6;
+}
+
+.col-three-fifth {
+  width: 60%;
+}
+
+.adjust-image-width-margin {
+  width: 150px;
+  margin-bottom: 40px;
+}
+
+.right-content {
+  display: flex;
+  color: white;
+  justify-content: space-between;
+  line-height: 2;
+}
+
+.links {
+  background-color: #fafafa;
+  padding-top: 30px;
+  padding-bottom: 30px;
+}
+
+.backgroundimg-producthead {
+  background-image:
+    linear-gradient(rgb(0 0 0 / 20%), rgb(0 0 0 / 30%)),
+    url("images/product/product-header.jpg");
+  background-size: cover;
+  background-repeat: no-repeat;
+  margin-left: auto;
+}
+
+.adjust-font-color-weight-product {
+  color: white;
+  font-weight: normal;
+  padding-top: 40px;
+}
+
+.adjust-font-color-size-family-product {
+  color: white;
+  line-height: 1.6;
+  font-family: Avenir, sans-serif;
+  font-size: 12px;
+  padding-bottom: 80px;
+}
+
+.product-img-left {
+  width: 100%;
+  padding-right: 80px;
+}
+
+.product-img-right {
+  width: 100%;
+  padding-left: 80px;
+}
```

Comment:
These seem like they should just be some padding utility classes added to a `column-full` element